### DISCUSSION TITLE
Can we use @OptionalArg and @Join in one argument?

### DISCUSSION THREAD

**Discussion by xiaoACE6716** [Rollczi/LiteCommands#406] [Q&A] [answered]
just like `public void onExecute(@Context CommandSender sender, @Context Message message, @Arg String atRole, @OptionalArg("description") @OptionalArg @Join String description) `
I want this description to be an optional parameter, and it can also be spliced together.

**Comment by vLuckyyy** [comment, marked as answer]
`@Join Optional<String> reason`
